src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -158,15 +192,17 @@ public function export(array $table)
         if ($result['rows'] > 0) {
             $this->createManifest($table);
         } else {
-            $this->logger->warn(sprintf(
-                "Query returned empty result. Nothing was imported for table [%s]",
-                $table['name']
-            ));
+            $this->logger->warn(
```

Comment:
This is probably some Code Style settings. I'd check this:
![image](https://user-images.githubusercontent.com/642928/39520450-f1ba7ca8-4e0a-11e8-8098-ca134b9b7c3f.png)